CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.0
+  * Added a workaround for an old analyzer error. It goes away with ddk (new
+    analyzer versions).
```

Comment:
[nit] It isn't a new analyzer version - it's switching to CFE away from analyzer. I'd phrase this as.

"Added a workaround for a bug in DDC (used in build_web_compilers 1.x). This bug is not present in DDK (used in build_web_compilers 2.x)"

Reply:
done